docker/red-mongodb/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM python:3.7-slim-stretch
+
+WORKDIR /app
+ENV LD_LIBRARY_PATH /usr/local/lib:/usr/lib
+
+RUN apt-get update && \
+    apt-get dist-upgrade -y && \
+    apt-get install -y --no-install-recommends git ca-certificates xz-utils libssl-dev build-essential && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* && \
+    pip install -U pip Cython pytest -q --retries 30 && \
```

Comment:
I think that 3 retries are enough or the container build could be blocked for 1 or 2 minutes before it fails.